README.md

```diff
@@ -79,9 +79,9 @@ $ make
 
 | Fedora / RedHat       | Ubuntu / Debian        | Comment                      |
 | --------------------- | ---------------------- |:---------------------------- |
-| `glibc-devel.i686`    | `libc6-dev:i386`       |                              |
+| `glibc-devel.i686`    | `libc6-dev-i386`       |                              |
 | `libgcc.i686`         | `gcc-multilib`         |                              |
-| `readline-devel.i686` | `libreadline-dev:i386` | Optional, used in mpscript.  |
+| `readline-devel.i686` | `lib32readline-dev`    | Optional, used in mpscript.  |
```

Comment:
I'm not really sure about this. The existing ones seem like the correct deps on my test system, but I'm not a Ubuntu expert (Fedora guy). Do you know what's different on our systems?